### PR TITLE
Probe external compasses on foxeer reaper f745

### PR DESCRIPTION
**Pull request by andyp1per**

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/ReaperF745/hwdef.dat [hunk 1]
```diff
@@ -126,6 +126,7 @@ SPIDEV osd       SPI1 DEVID1 AT7456E_CS MODE0 10*MHZ 10*MHZ
 
 # no built-in compass and no external I2C so no compass
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/ReaperF745/hwdef.dat:127]: So I guess this comment might be out of date now?

Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/ReaperF745/hwdef.dat:127]: I think just a copy and paste error

**Review by tridge — APPROVED** [review]

**Comment by andyp1per** [comment]
Thanks @tridge !

**Comment by rmackay9** [comment]
This is included in Copter-4.3.4-rc1

**Pull request merged**